src/platform/amd/renoir/include/platform/chip_registers.h

```diff
@@ -0,0 +1,571 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2021 AMD.All rights reserved.
+ *
+ * Author:      Basavaraj Hiregoudar <basavaraj.hiregoudar@amd.com>
+ *              Anup Kulkarni <anup.kulkarni@amd.com>
+ *              Bala Kishore <balakishore.pati@amd.com>
+ */
+#if !defined(_rn_REG_HEADER)
+#define _rn_REG_HEADER
+
+typedef	union acp_dma_cntl_0 {
+	struct {
+	unsigned int	dmachrst:1;
+	unsigned int	dmachrun:1;
+	unsigned int	dmachiocen:1;
+	unsigned int	: 29;
+	} bits;
+unsigned int	u32all;
+} acp_dma_cntl_0_t;
```

Comment:
the indentation seems off in this file?

Reply:
oh... automatically generated? Maybe the script can be improved a bit ;-)

Reply:
Looks like this is generated, so it should also come under the same style exception as the Cadence xtensa headers.